storage-blobs-python-quickstart-v4/example.py

```diff
@@ -0,0 +1,102 @@
+# This sample application creates a test file, uploads the test file to the Blob storage,
+# lists the blobs in the container, and downloads the file with a new name.
+# ---------------------------------------------------------------------------------------------------------
+# Documentation References:
+# Associated Article - https://docs.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-python
+# What is a Storage Account - http://azure.microsoft.com/en-us/documentation/articles/storage-whatis-account/
+# Getting Started with Blobs-https://docs.microsoft.com/en-us/azure/storage/blobs/storage-python-how-to-use-blob-storage
+# Blob Service Concepts - http://msdn.microsoft.com/en-us/library/dd179376.aspx
+# Blob Service REST API - http://msdn.microsoft.com/en-us/library/dd135733.aspx
+# ----------------------------------------------------------------------------------------------------------
+
+
+def run_sample():
+    try:
+        # Create the BlobServiceClient that is used to call the Blob service for the storage account
+        block_blob_service = BlobServiceClient.from_connection_string(conn_str="Connection string")
+
+        # Create a container called 'quickstartblobs' and Set the permission so the blobs are public.
+        container_name ='quickstartblobs'
+        block_blob_service.create_container(container_name, public_access=PublicAccess.Container)
+
+        # Create a file in Documents to test the upload and download.
+        local_path=os.path.expanduser("~/Documents")
+        local_file_name ="QuickStart_" + str(uuid.uuid4()) + ".txt"
+        full_path_to_file =os.path.join(local_path, local_file_name)
+
+        # Write text to the file.
+        file = open(full_path_to_file,  'w')
+        file.write("Hello, World!")
+        file.close()
+
+        print("Temp file = " + full_path_to_file)
+        print("\nUploading to Blob storage as blob" + local_file_name)
+
+        # Upload the created file, use local_file_name for the blob name
+        blob_client = BlobClient.from_connection_string("Connection string", container=container_name, blob=local_file_name)
+        with open(full_path_to_file, "rb") as data:
+            blob_client.upload_blob(data)
+
+        # List the blobs in the container
+        print("\nList blobs in the container")
+        container = ContainerClient.from_connection_string("Connection string", container=container_name)
+        generator = container.list_blobs()
+        for blob in generator:
+            print("\t Blob name: " + blob.name)
+
+        # Download the blob(s).
+        # Add '_DOWNLOADED' as prefix to '.txt' so you can see both files in Documents.
+        full_path_to_file2 = os.path.join(local_path, str.replace(local_file_name ,'.txt', '_DOWNLOADED.txt'))
+        print("\nDownloading blob to " + full_path_to_file2)
+        with open(full_path_to_file2, "wb") as my_blob:
+            my_blob.writelines(blob_client.download_blob())
```

Comment:
This will be
`my_blob.writelines(blob_client.download_blob().readall())` in the latest version. If you are pointing to the preview2,
it will be `my_blob.writelines(blob_client.download_blob().content_as_bytes())`